src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala

```diff
@@ -30,34 +30,34 @@ class PolicyEvaluatorService(
           Set.empty,
           Set(SamResourceActions.setPublicPolicy(ManagedGroupService.adminNotifierPolicyName)),
           true
-        ))
+        ), SamRequestContext(None))
```

Comment:
ditto internal trace

Reply:
Intellij says this is only called on Boot. Do you think this call (and the other similar 1-time init/boot calls) are worth tracing as well? I was leaning toward no but am curious to know other thoughts.

Reply:
Probably not super interesting to trace boot functions (though would be good if we try to optimize start up time), so seems fine to leave for later.
The only reason to care more deeply is if they seem likely to be used additional times.
Let's just leave a comment about why we're not bothering to trace it for now.